src/main/java/org/folio/circulation/domain/Loan.java

```diff
@@ -352,12 +352,12 @@ public Loan overrideRenewal(DateTime dueDate,
     return this;
   }
 
-  Loan checkIn(DateTime returnDate, UUID servicePointId) {
+  Loan checkIn(DateTime returnDate, DateTime systemDateTime, UUID servicePointId) {
```

Comment:
I believe it is the system return date time. I'm happy for DateTime elements to be named with the `DateTime` suffix. I think we need to be consistent, please change the `returnDate` as well.

```suggestion
  Loan checkIn(DateTime returnDateTime, DateTime systemReturnDateTime, UUID servicePointId) {
```

Reply:
Done.

Reply:
Done.